app/websocket_app.rb

```diff
@@ -272,33 +275,55 @@ class WebsocketApp
     end
   end
 
+  def update_node_attributes(node_name, message)
+    node = FlightScheduler.app.default_partition.nodes.find do |n|
+      n.name == node_name
+    end
+    return unless node
+
+    attributes = node.attributes.dup
+    attributes.cpus   = message[:cpus]    if message.key?(:cpus)
+    attributes.gpus   = message[:gpus]    if message.key?(:gpus)
+    attributes.memory = message[:memory]  if message.key?(:memory)
+
+    if attributes.valid?
+      node.attributes = attributes
```

Comment:
Could we get some logging either here or in the node connected log to show the attributes that have been set for it?

Reply:
Logging added